src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -0,0 +1,676 @@
+	 * @return array The current state for the specified store namespace. This will be the updated state if a $state
+	 *               argument was provided.
+	 */
+	public function state( string $store_namespace, array $state = array() ): array {
+		if ( ! isset( $this->state_data[ $store_namespace ] ) ) {
+			$this->state_data[ $store_namespace ] = array();
+		}
+		if ( is_array( $state ) ) {
+			$this->state_data[ $store_namespace ] = array_replace_recursive(
+				$this->state_data[ $store_namespace ],
+				$state
+			);
+		}
+		return $this->state_data[ $store_namespace ];
+	}
+
+	/**
+	 * Gets and/or sets the configuration of the Interactivity API for a given
+	 * store namespace.
+	 *
+	 * If configuration for that store namespace exists, it merges the new
+	 * provided configuration with the existing one.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param string $store_namespace The unique store namespace identifier.
+	 * @param array  $config          Optional. The array that will be merged with the existing configuration for the
+	 *                                specified store namespace.
+	 * @return array The configuration for the specified store namespace. This will be the updated configuration if a
+	 *               $config argument was provided.
+	 */
+	public function config( string $store_namespace, array $config = array() ): array {
+		if ( ! isset( $this->config_data[ $store_namespace ] ) ) {
+			$this->config_data[ $store_namespace ] = array();
+		}
+		if ( is_array( $config ) ) {
+			$this->config_data[ $store_namespace ] = array_replace_recursive(
+				$this->config_data[ $store_namespace ],
+				$config
+			);
+		}
+		return $this->config_data[ $store_namespace ];
+	}
+
+	/**
+	 * Prints the serialized client-side interactivity data.
+	 *
+	 * Encodes the config and initial state into JSON and prints them inside a
+	 * script tag of type "application/json". Once in the browser, the state will
+	 * be parsed and used to hydrate the client-side interactivity stores and the
+	 * configuration will be available using a `getConfig` utility.
+	 *
+	 * @since 6.5.0
+	 */
+	public function print_client_interactivity_data() {
+		$store      = array();
+		$has_state  = ! empty( $this->state_data );
+		$has_config = ! empty( $this->config_data );
+
+		if ( $has_state || $has_config ) {
+			if ( $has_config ) {
+				$store['config'] = $this->config_data;
+			}
+			if ( $has_state ) {
+				$store['state'] = $this->state_data;
+			}
+			wp_print_inline_script_tag(
+				wp_json_encode(
+					$store,
+					JSON_HEX_TAG | JSON_HEX_AMP
+				),
+				array(
+					'type' => 'application/json',
+					'id'   => 'wp-interactivity-data',
+				)
+			);
+		}
+	}
+
+	/**
+	 * Registers the `@wordpress/interactivity` script modules.
+	 *
+	 * @since 6.5.0
+	 */
+	public function register_script_modules() {
+		$suffix = wp_scripts_get_suffix();
+
+		wp_register_script_module(
+			'@wordpress/interactivity',
+			includes_url( "js/dist/interactivity{$suffix}.js" )
```

Comment:
My suggestion should have omitted the curly braces.

```suggestion
			includes_url( "js/dist/interactivity$suffix.js" )
```

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/5953/commits/ee6e780a3720d65f88db60f187847ca36aa79255.